### PR TITLE
miller 5.1.0

### PR DESCRIPTION
**Pull request by johnkerl**
Next release following same process as always

### CODE REVIEW

File: Formula/miller.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Miller < Formula
   desc "Like sed, awk, cut, join & sort for name-indexed data such as CSV"
   homepage "https://github.com/johnkerl/miller"
-  url "https://github.com/johnkerl/miller/releases/download/v5.0.1/mlr-5.0.1.tar.gz"
-  sha256 "64d9c16bf9d6ef45f22ef43a6652f784ecd2b629004f03dfd1c6dd9572377150"
+  url "https://github.com/johnkerl/miller/releases/download/v5.1.0/mlr-5.0.1.tar.gz"
```
Comment (neutric) [inline Formula/miller.rb]: Please update the latter part of the url as well.

**Review by neutric — CHANGES_REQUESTED** [review]

File: Formula/miller.rb (hunk 1 above)
Comment (johnkerl) [inline Formula/miller.rb]: Thank you! On it.

**Comment by johnkerl** [comment]
Fixed

**Review by neutric — APPROVED** [review]

**Comment by neutric** [comment]
@johnkerl Thanks for keeping `miller` updated!

**Pull request closed without merging**